apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( 'a8c/navigation-menu', {
+	title: __( 'Navigation Menu' ),
+	description: __( 'Visual placeholder for site-wide navigation and menus.' ),
+	icon: 'layout',
+	category: 'layout',
+	supports: {
+		html: false,
+		multiple: false,
+		reusable: false,
+	},
+	attributes: {
+		themeLocation: {
+			type: 'string',
+			default: 'main-1',
+		},
+	},
+	edit,
+	save: () => null,
```

Comment:
With a no-op on save, this means that only `<!-- wp:a8c/navigation-menu /-->` will be saved.

We can follow up on this in another PR but we probably do want to at least allow `themeLocation` to be parsed and serialized to allow a primary/social logo to be in the template_part "header".

Reply:
Following up on this in testing, it looks like we can have a no-op for save, for standard block comment attributes.

Eg: `<!-- wp:a8c/navigation-menu {"themeLocation":"social"} /-->` appears to be saved / loaded okay on page refresh 👍

Reply:
Cool, that was my conclusion too 🎉